### ISSUE TITLE
[Fluid-DEM] not running

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/Kratos#5458]
In the interface example:

```
Traceback (most recent call last):
  File "MainKratos.py", line 32, in <module>
    simulation = SwimmingDEMAnalysisWithFlush(model=model)
  File "MainKratos.py", line 22, in __init__
    super(SwimmingDEMAnalysisWithFlush, self).__init__(model, parameters)
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\applications\SwimmingDEMApplication\python_scripts\swimming_DEM_analysis.py", line 79, in __init__
    self._GetFluidAnalysis().coupling_analysis = weakref.proxy(self)
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\applications\SwimmingDEMApplication\python_scripts\swimming_DEM_analysis.py", line 736, in _GetFluidAnalysis
    import DEM_coupled_fluid_dynamics_analysis as fluid_analysis
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\applications\SwimmingDEMApplication\python_scripts\DEM_coupled_fluid_dynamics_analysis.py", line 6, in <module>
    from fluid_dynamics_analysis import FluidDynamicsAnalysis
ModuleNotFoundError: No module named 'fluid_dynamics_analysis'
```

and the application tests: 
```
PS E:\PROYECTOS\Kratos\applications\SwimmingDEMApplication\tests> runkratos.exe .\SmallTests.py
 |  /           |
 ' /   __| _` | __|  _ \   __|
 . \  |   (   | |   (   |\__ \
_|\_\_|  \__,_|\__|\___/ ____/
           Multi-Physics 7.0.0-7fafdb9b31-Release
Compiled with OpenMP support.
Maximum OpenMP threads: 8.
Traceback (most recent call last):
  File ".\SmallTests.py", line 4, in <module>
    import TestFactory as TF
  File "E:\PROYECTOS\Kratos\applications\SwimmingDEMApplication\tests\TestFactory.py", line 9, in <module>
    from tests_python_scripts.candelier_scripts.candelier_analysis import CandelierBenchmarkAnalysis
  File "E:\PROYECTOS\Kratos\applications\SwimmingDEMApplication\tests\tests_python_scripts\candelier_scripts\candelier_analysis.py", line 3, in <module>
    import swimming_DEM_procedures as SDP
ModuleNotFoundError: No module named 'swimming_DEM_procedures'
KRATOS TERMINATED WITH ERROR
```

**Comment by philbucher** [comment]
#5268

**Comment by maceligueta** [comment]
Thanks for the report @jginternational . This is being fixed in #5475 .

**Comment by philbucher** [comment]
closing since I believe this is fixed
